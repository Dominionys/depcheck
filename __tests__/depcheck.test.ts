const {depcheck} = require('../index.js');

test('should not failed', () => {
  const result = depcheck('./crates/core/tests/fake_modules/bad');
  expect(result).toMatchSnapshot();
});
