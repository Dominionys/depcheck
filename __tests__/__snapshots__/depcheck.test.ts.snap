// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not failed 1`] = `
Object {
  "missingDependencies": Object {},
  "unusedDependencies": Array [
    "optimist",
  ],
  "unusedDevDependencies": Array [],
  "usingDependencies": Object {},
}
`;
